types/stderror.go

```diff
@@ -28,7 +28,7 @@ var (
 	_ error = ParseErr{}
 	_ error = SerializeErr{}
 	_ error = Unauthorized{}
-	_ error = UnderflowErr{}
+	_ error = Underflow{}
```

Comment:
NullPointer is missing in this list and `ToStdError`

Reply:
Good point, I will add it.

Reply:
Done